root/defaults/nginx/authelia-server.conf.sample

```diff
@@ -62,11 +33,6 @@ location @authelia_proxy_signin {
     ## Translate the Location response header from the auth subrequest into a variable
     auth_request_set $signin_url $upstream_http_location;
 
```

Comment:
i **think** you need the below, at least I have this in mine
```
    if ($signin_url = '') {
        ## Set the $signin_url variable
        set $signin_url https://$upstream_authelia/authelia/auth-request;
    }
```

https://www.authelia.com/reference/guides/proxy-authorization/#authrequest
and
https://www.authelia.com/blog/4.38-release-notes/
`For example if your previous proxy configuration included http://authelia:9091/api/verify?rd=https://auth.example.com it now by default becomes http://authelia:9091/api/authz/forward-auth for Traefik / Caddy / HAProxy, by default becomes http://authelia:9091/api/authz/auth-request for NGINX based proxies, and by default becomes http://authelia:9091/api/authz/ext-authz for Envoy.`

Reply:
This was removed because `signin_url` should always be set via header response from authelia in 4.38. If it somehow does not get set that would mean authelia didn't send it and the auth has failed in some way.

Having `set $signin_url https://$upstream_authelia/authelia/auth-request;` would need authelia configured to respond with the subpath `/authelia`, which we're removing comment instructions to configure since that seemed to be a complication with users trying to set up authelia.

Reply:
I do not have the subpath set and what I showed works fine, we will definitely need to test because I'm pretty sure the 4.38 migration guide says it's needed (I think I linked but I'm on mobile)